app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -78,7 +78,8 @@
     var upload_table = document.getElementById("files-to-upload");
 
     var filesUploadButton = document.getElementById("upload-button");
-    var txt = "";
+
+    $(upload_table).find("tr>td").remove();
```

Comment:
The user _can_ upload multiple files at once if they select multiple files in a single file-select dialog box.

Perhaps we should add a tooltip that lets them know they can do this (so if they _do_ want to upload multiple files it is less frustrating - although we also need to fix this eventually so they can upload multiple files by clicking the "Choose files ..." button multiple times).

Here if the form when multiple files are specified in the dialog box:

<img width="769" alt="screen shot 2017-12-04 at 6 23 23 pm" src="https://user-images.githubusercontent.com/9968213/33581994-0fb92ed4-d921-11e7-8bff-ed54db27c6d1.png">

And here is the result of clicking the "save" button:

<img width="301" alt="screen shot 2017-12-04 at 6 23 58 pm" src="https://user-images.githubusercontent.com/9968213/33582005-22762126-d921-11e7-98b6-a8f321cb2699.png">

Reply:
Good idea with a Tooltip, can we have it in and shown at the client meeting and then perhaps deployed after?

Reply:
The tooltip (or she other "help" mechanism) should explain how to select multiple files on a Mac and on windows.

Eventually, we should redesign the file upload (sub)form to allow the user to add another file to upload (as many times as they want), using, perhaps, the cocoon gem (which works very well for this kind of situation).  (https://github.com/nathanvda/cocoon)

Reply:
What about doing some simple text like this for now:  
- Add text that tells them they can upload multiple files at once (_"You can upload multiple files..."_  in my rough example below)
- Add explicit instructions for Mac and PC below that.

---
<img width="754" alt="multi-file upload instructions idea" src="https://user-images.githubusercontent.com/673794/33627890-cc6401b6-d9b3-11e7-87aa-95c779c49178.png">


---
That way it is not hidden in a tooltip.  
If people using this are not power-users, then they need instructions to stay visible so they can follow the steps as they do something.  When it's in a pop-up tooltip, they can't keep referring to it easily.  (Pop-up tootips are good for quick reminders)

(I am assuming that most members are _not_ "power users".)

Reply:
I like this idea a lot better than a tooltip :)